### PR TITLE
Basic inspector for V2 physics

### PR DESCRIPTION
**Pull request by carolhmj**
Related to [#13711](https://github.com/BabylonJS/Babylon.js/issues/13711)
This adds a basic interface for physics properties on the inspector, following the same style as V1 physics. One possible evolution would be to display physics bodies and shapes in their own entity tree, instead of being a property of the mesh.
image

When the body is instanced, it is possible to select an instance through the first line:
image

Only the mass is editable, because friction/restitution are properties of a shape, which can be shared between different bodies, so changing it could affect other bodies.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14718/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14718/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14718/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14718/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14718/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14718/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14718/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14718/merge#BCU1XR#0

File: packages/dev/core/src/Physics/v2/physicsBody.ts [hunk 1]
```diff
@@ -62,6 +62,10 @@ export class PhysicsBody {
 
     private _isDisposed = false;
 
+    private _shape: Nullable<PhysicsShape> | undefined;
```
Comment (RaananW) [inline packages/dev/core/src/Physics/v2/physicsBody.ts:65]:
```suggestion
    private _shape: Nullable<PhysicsShape> = null;
```

File: packages/dev/core/src/Physics/v2/physicsBody.ts [hunk 2]
```diff
@@ -160,15 +166,25 @@ export class PhysicsBody {
...
     /**
      * Sets the shape of the physics body.
      * @param shape - The shape of the physics body.
      *
      * This method is useful for setting the shape of the physics body, which is necessary for the physics engine to accurately simulate the body's behavior.
      * The shape is used to calculate the body's mass, inertia, and other properties.
      */
-    public set shape(shape: Nullable<PhysicsShape>) {
-        this._physicsPlugin.setShape(this, shape);
+    public set shape(shape: Nullable<PhysicsShape> | undefined) {
```
Comment (RaananW) [inline packages/dev/core/src/Physics/v2/physicsBody.ts:183]:
```suggestion
    public set shape(shape: Nullable<PhysicsShape> = null) {
```

File: packages/dev/core/src/Physics/v2/physicsBody.ts [hunk 3]
```diff
@@ -180,8 +196,8 @@ export class PhysicsBody {
      * This method is useful for retrieving the physics shape associated with this object,
      * which can be used to apply physical forces to the object or to detect collisions.
      */
-    public get shape(): Nullable<PhysicsShape> {
-        return this._physicsPlugin.getShape(this);
+    public get shape(): Nullable<PhysicsShape> | undefined {
```
Comment (RaananW) [inline packages/dev/core/src/Physics/v2/physicsBody.ts:199]:
```suggestion
    public get shape(): Nullable<PhysicsShape> {
```

**Review by RaananW — APPROVED** [review]
just a nit! (not a big fan of `Nullable<Type> | undefined`)

**Review by CedricGuillemet — APPROVED** [review]
This is so great!

**Pull request merged**